app/views/tutorials/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<%- @page_title = 'Tutorials' -%>
+
+<div class="usa-grid usa-section usa-layout-docs">
+  <%= render 'layouts/sidenav' %>
+
+  <div class="usa-width-three-fourths usa-layout-docs-main_content">
+    <h1><%= @page_title %></h1>
+
+    <p>Defense Personal Property System (DPS) is an online application where service members and <%= abbr_tag('dod') %> civilians can schedule their household goods move (also known as “self-counseling”) and submit any loss or damage claims directly to their Transportation Service Provider (TSP).</p>
+    <p><b>Use the tutorials below for help navigating <%= abbr_tag('dps') %>.</b> If you get stuck or have any questions, you can visit your <%= link_to 'local transportation office', '#TODO' %> or call the 24-hour <%= abbr_tag('src') %> Help Desk at <%= link_to '1 (800) 462-2176', 'tel:+18004622176' %>.</p>
```

Comment:
Right... but I guess my point is that if someone comes along and tells you it shouldn't be bold, but red then you need to change two things: the tag and the css (because `<b>` would no longer be valid). If you use `<strong>` instead, and they decide it should be red and not bold, then you only change the css. This is more philosophical, and it's not my project, but I always err to the side of `<strong>` for the reason above.